sdn_tests/testplans/SUBSCRIBE-Modes-Levels-Test-Plan.md

```diff
@@ -0,0 +1,2150 @@
+# Overview
+
+This document aims to outline the approach for testing the functionality of different SUBSCRIBE modes of the gNMI protocol as part of GPINs OpenConfig end-to-end testing. Subscriptions in gNMI can be periodic or event driven. Modifiers can also be applied to request heartbeat updates or filter out unchanged values.
```

Comment:
This document aims to outline the approach for testing the functionality of different SUBSCRIBE modes of the gNMI protocol as part of OpenConfig end-to-end testing. Subscriptions in gNMI can be periodic or event driven. Modifiers can also be applied to request heartbeat updates or filter out unchanged values.